docs/reference/hooks.rst

```diff
@@ -533,6 +533,16 @@ Hooks for customising the way users are directed through the process of creating
 
   Uses the same behaviour as ``before_create_page``.
 
+ .. code-block:: python
+
+    from wagtail.core import hooks
+
+    from .models import AwesomePage
+    from .admin_views import edit_awesome_page
+
+    @hooks.register('before_delete_page')
+    def before_delete_page(request, page):
+        return delete_awesome_page(request, page)
```

Comment:
I know this is also in the `before_create_page` example, but I'm not sure why we import `edit_awesome_page` then call `delete_awesome_page`.

Reply:
Yeah I was actually wondering about that as well, but I am not super familiar with the project so I figured I just kept it. Happy to remove it in both of the examples though!